.eslint-dictionary.json

```diff
@@ -36,12 +36,14 @@
   "frontend",
   "func",
   "generatable",
+  "getschemadeployer",
```

Comment:
nit: can we ignore spellcheck for some lines and/or camelCase usages of these strings rather than add word concatenations to the dictionary?